2023-12-17-CrypoGo.md

tbpatil opened the pull request:
Added Blog Page for CrypGo.

tbpatil committed:
CrypGo Blog

cjcchen commented:
Very nice blog.

After going through the details, it would be better to add more commands to show how to use each function.
Could you add some?

DakaiKang reviewed (changes requested):
Hi Toniya,

Please change the name of your file to "2023-12-xx-CrypoGo.md", in which xx is the day you created the blog. Also, please add some screenshots of your application, showing the functionalities of your app. Thank you. Good job.

tbpatil committed:
Udpate and rename 

Added Screenshots of features and renamed the file according to format. (2023-12-27-CrypoGo.md)

DakaiKang reviewed (approved):
Changes needed. But no worries, I will finish these.
